app/controllers/events/view/speakers/list.js

```diff
@@ -15,7 +15,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
       {
         name            : 'Name',
         valuePath       : 'name',
-        extraValuePaths : ['id', 'event'],
+        extraValuePaths : ['id'],
```

Comment:
But this won't work as I said this.extraRecords is not complete speaker object, it has prop of speaker object that are defined in extraValuePaths.

Reply:
@iamareebjamal its not working with this change, please see to it.

Reply:
OK, please send a PR with this change

Reply:
OK and specify same issue #4559 with it?

Reply:
Same

Reply:
Please see PR #5405